experiments/AMIP/moist_mpi_earth/coupler_driver_modular.jl

```diff
@@ -320,7 +324,7 @@ cs = CoupledSimulation{FT}(
     coupler_fields,
     parsed_args,
     conservation_checks,
-    tspan,
+    [tspan[1], tspan[2]],
```

Comment:
would it be simpler to initialize `tspan` as an array of length 2 instead? i.e. `tspan = [Int(0), t_end]`, then we can just pass in `tspan` here.

Reply:
Doing it this way allows specifying tspan as a Tuple (like in Atmos's global arguments which overwrites our tspan) or as an array. I know... and I agree that we should unify this, once we don't have synonymous global variables lingering about in the background. :)